test/controllers/TransferSummaryControllerSpec.scala

```diff
@@ -151,11 +151,83 @@ class TransferSummaryControllerSpec extends FrontEndTestHelper {
       val controller = new TransferSummaryController(getAuthorisedSecurityComponents, new GraphQLConfiguration(app.configuration),
         getValidKeycloakConfiguration, consignmentService, exportService(app.configuration), langs)
       val finalTransferConfirmationSubmitResult = controller.finalTransferConfirmationSubmit(consignmentId)
-        .apply(FakeRequest(POST, s"/consignment/$consignmentId/transfer-summary").withCSRFToken)
+        .apply(FakeRequest(POST, s"/consignment/$consignmentId/transfer-summary")
+          .withFormUrlEncodedBody(finalTransferConfirmationForm(openRecordsValue = false, transferLegalOwnershipValue = false): _*)
+          .withCSRFToken)
+
+      playStatus(finalTransferConfirmationSubmitResult) mustBe BAD_REQUEST
+      contentAsString(finalTransferConfirmationSubmitResult) must include("govuk-error-message")
+      contentAsString(finalTransferConfirmationSubmitResult) must include("error")
+      contentAsString(finalTransferConfirmationSubmitResult) must include("There is a problem")
+      contentAsString(finalTransferConfirmationSubmitResult) must include("#error-openRecords")
+      contentAsString(finalTransferConfirmationSubmitResult) must include("#error-transferLegalOwnership")
+      contentAsString(finalTransferConfirmationSubmitResult) must include("transferSummary.openRecords.error")
+      contentAsString(finalTransferConfirmationSubmitResult) must include("transferSummary.transferLegalOwnership.error")
+    }
+
+    "display correct error when only the open records option is selected and the final transfer confirmation form is submitted" in {
+      val client = new GraphQLConfiguration(app.configuration).getClient[gcs.Data, gcs.Variables]()
```

Comment:
You could abstract the setup of the test into a separate function, as it is the same for both the tests